maps/away/away_site/pirate_base/pirate_base.dm

```diff
@@ -15,12 +15,7 @@
 
 /obj/effect/overmap/visitable/sector/pirate
 	name = "asteroid lair"
-	desc = "\
-		Scans reveal that there is a unregistered structure within this asteroid, \
-		as well as an unpowered vessel docked in a makeshift hangar on the south outer layer. \
-		Further examination reveals it is wanted in the Coalition of Colonies and that its crew have a bounty on them for piracy and kidnapping. \
-		Extreme caution is advised.\
-		"
+	desc = "Scans reveal that there is a unregistered structure within this asteroid, as well as an unpowered vessel docked in a makeshift hangar on the south outer layer."
```

Comment:
> Changes the pirate base description to not immediately tell anyone scanning it that there are pirates there so that the Horizon can doesn't have a massive IC reason to avoid it.

This was intentional, and it should stay. The pirate base is not a normal away site, and should not be treated as one. It has actual antagonists, so the description when scanned should warn Horizon to be careful and bring armed security to check it out. It is specifically intended to be a site that Horizon can visit if they want to fight the pirates, being justified and able to actually prepare for that fight.

I do not want people going there unarmed and without security, and having a bad experience being killed or ganked or kidnapped by pirates who have greater numbers and better gear. And I do not want people metagaming that this random asteroid outpost is a definitely a pirate base so we should totally bring security and guns to explore it.

Reply:
The only thing I would change, is to change it from "wanted in the Coalition of Colonies" to "wanted in the nearest governing body" or something. And just make it "Caution is advised" without the "extreme".

Reply:
Okay well, the warning could be something else, and not say literally "there are nasty pirates there". But it should still be a warning that says the site may be dangerous.